### ISSUE TITLE
Quick cluster test by using pretrained model？

### ISSUE THREAD

**Issue by dejianchen-x** [yukimasano/self-label#11]
Is there a convenient way to quickly cluster on some messy images, if I only use the pretrained model？

**Comment by yukimasano** [comment]
Hi there,
Yes, I believe that should work well.
e.g. try
```
import torchvision, torch
model = torchvision.models.resnet18(pretrained=True)
model.fc = torch.nn.Linear(512, NCL)
```
where `NCL` would be the number of cluster you want to cluster into. Perhaps you can also just backprop gradients to the last couple of layers. Let me if/how it works!
